src/sammich/plugins/project.py

```diff
@@ -1,32 +1,90 @@
         super().__init__(client, settings, storage)
-        with open("data/projects.json") as f:
-            self.project_data = json.load(f)
+
+        try:
+            with open("data/projects.json") as f:
+                self.project_data = json.load(f)
+        except json.JSONDecodeError:
+            self.project_data = {}
 
     @command("/project")
     async def project(self, command):
-        text = command.text.strip()
-        project_name = text.strip().lower()
+        project_name = command.text.strip().lower()
+        channel_id = command._cmd_payload["channel_id"]
 
         project = self.project_data.get(project_name)
 
         if project:
             project_list = "\n".join(project)
             message = f"Hello, here the information about '{project_name}':\n{project_list}"
+            await command.say(message)
         else:
-            message = (
-                f"Hello, the project '{project_name}' is not recognized. "
-                "Please try different query."
+            # if the self.project_data is empty, return a message
+            if not self.project_data:
+                await command.say("No projects available")
+                return None
+            await self.show_project_page(channel_id)
+
+    async def show_project_page(self, channel_id):
+        projects = list(self.project_data.keys())
+
+        # Calculate the number of dropdowns needed
+        num_dropdowns = (len(projects) + PROJECTS_PER_PAGE - 1) // PROJECTS_PER_PAGE
+
+        blocks = []
+        project_for_each_page = 75
+        for i in range(num_dropdowns):
+            options = [
+                {
+                    "text": {"type": "plain_text", "text": project[:project_for_each_page]},
+                    "value": project,
+                }
+                for project in projects[
+                    i * PROJECTS_PER_PAGE : i * PROJECTS_PER_PAGE + PROJECTS_PER_PAGE
+                ]
+            ]
+
+            blocks.append(
+                {
+                    "type": "section",
+                    "block_id": f"project_select_block_{i}",
+                    "text": {
+                        "type": "mrkdwn",
+                        "text": f"Select a project (Page {i + 1}):",
+                    },
+                    "accessory": {
+                        "type": "static_select",
+                        "placeholder": {
+                            "type": "plain_text",
+                            "text": f"Select a project (Page {i + 1})",
+                        },
+                        "options": options,
+                        "action_id": f"project_select_action_{i}",
+                    },
+                }
             )
 
-        await command.say(message)
+        await self.web_client.chat_postMessage(
+            channel=channel_id, blocks=blocks, text="Available Projects"
+        )
```

Comment:
Can we use `.say()` here?

Reply:
basically sir ,I was trying to use .say() . but the issue , we can't , cause the block has some action and action are not allowed for .say() , I mean where the "is_ephemeral" is true. if we use .say() , the action not gonna work anymore

Reply:
Okay, could you leave a comment around L80 describing the issue then?

Thank you!

Reply:
ok sure